chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -565,7 +565,7 @@ void BraveRenderViewContextMenu::ExecuteAIChatCommand(int command) {
                        ->GetDefaultAIEngine();
     }
     ai_engine_->GenerateRewriteSuggestion(
-        selected_text, ai_chat::GetActionTypeQuestion(action_type),
+        selected_text, ai_chat::GetActionTypeQuestion(action_type), "",
```

Comment:
Nit: Would be nice to add /*selected_language*/ to the new param.

Reply:
Fixed in e31d368a2be0babfb209af4f8174745b360d7dc8